pulp_2to3_migration/app/plugin/docker/migrator.py

```diff
@@ -75,6 +76,10 @@ class DockerMigrator(Pulp2to3PluginMigrator):
         'docker_distributor_web': DockerDistributor,
     }
 
+    premigrate_hook = {
+        'docker_tag': utils.find_tags()
```

Comment:
I think this needs to store a reference to the function and not be executed here. 
Is your goal to store results of the call on the migrator or just to know which hook to call?
I think the latter but correct me if I'm wrong.

My suggestion is to have:
```
premigrate_hook = {
￼        'docker_tag': utils.find_tags
}
```